src/frontend/context/OppgaverContext.tsx

```diff
@@ -239,59 +238,35 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
         behandlingstema?: string,
         oppgavetype?: string,
         enhet?: string,
-        prioritet?: string,
         frist?: string,
         registrertDato?: string,
         saksbehandler?: string
     ): Promise<Ressurs<IOppgave[]>> => {
-        interface LooseObject {
-            [key: string]: string;
-        }
-        const searchParams: LooseObject = {};
-
-        if (limit !== undefined) {
-            searchParams['limit'] = limit.toString();
-        }
-
-        if (behandlingstema !== undefined) {
-            searchParams['behandlingstema'] = behandlingstema;
-        }
-
-        if (oppgavetype !== undefined) {
-            searchParams['oppgavetype'] = oppgavetype;
-        }
-
-        if (enhet !== undefined) {
-            searchParams['enhet'] = enhet;
-        }
-
-        if (prioritet !== undefined) {
-            searchParams['prioritet'] = prioritet;
-        }
-
-        if (registrertDato !== undefined) {
-            searchParams['registrertDato'] = registrertDato;
-        }
-
-        if (frist !== undefined) {
-            searchParams['frist'] = frist;
-        }
-
-        if (saksbehandler !== undefined) {
-            searchParams['saksbehandler'] = saksbehandler;
-        }
-
-        let query = new URLSearchParams(searchParams).toString();
-        query = query === '' ? query : '?' + query;
+        const finnOppgaveRequest: IFinnOppgaveRequest = {
+            behandlingstema: behandlingstema,
+            oppgavetype: oppgavetype,
+            enhet: enhet,
+            saksbehandler: saksbehandler,
+            journalpostId: undefined,
+            opprettetFomTidspunkt: `${registrertDato}T00:00:00.000`,
```

Comment:
Is this really neccasary? Can't we just create a moment object and run toIsoString()?

Reply:
You will still have to specify the full iso date for opprettetTomTidspunkt, though, like this: moment(`${registrertDato}T23:59:59.999`).toIsoString()

This will append a Z at the end of the string. It's ISO standard but I haven't tested if that works, so a heads up there.

Reply:
moment supports different formatting, so it is sufficient to say:
`moment(registrertDato, datoformat.DATO).toIsoString()`

See datoformat for the different formats and use the appropriate one.

Reply:
I think what Geirmund means is that the TOM date is not straight forward to generate.
But why do we have to use moment? It just complicates the logic.

Reply:
Ok, sure 👍